CHANGELOG.md

```diff
@@ -2,6 +2,58 @@
 
 ## Unreleased
 
+### API
+
+* Add `GlobalOpenTelemetry#getOrNoop`, `#isSet`. Update `#get` to always returns obfuscated instance
+  ([#7819](https://github.com/open-telemetry/opentelemetry-java/pull/7819))
```

Comment:
🎉

Reply:
Here's hoping we don't end up regretting it. We've resisted for so long!